### PR TITLE
Updates to tiles clients daily

### PR DESCRIPTION
**Pull request by skahmann3**
Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/telemetry_derived/sponsored_tiles_clients_daily_v1/query.sql [hunk 1]
```diff
@@ -189,149 +138,235 @@ android_metrics AS (
...
     FROM
-      `moz-fx-data-shared-prod.firefox_ios.clients_last_seen_joined`
+      `mozdata.firefox_ios.events_unnested`
     WHERE
-      submission_date = @submission_date
-  ) profile_age_data
+      DATE(submission_timestamp) = @submission_date
+    GROUP BY
+      client_info.client_id
+  ) experiments_info
```
Comment (curtismorales) [inline sql/moz-fx-data-shared-prod/telemetry_derived/sponsored_tiles_clients_daily_v1/query.sql:281]: This isn't an issue for this PR, but I wonder if this same logic is being done in other tables and we should just make `experiments_info` its own derived table

File: sql/moz-fx-data-shared-prod/telemetry_derived/sponsored_tiles_clients_daily_v1/query.sql [hunk 2]
```diff
@@ -1,93 +1,166 @@
...
+    sample_id,
+    SUM(sponsored_tiles_click_count) AS sponsored_tiles_click_count
   FROM
-    `mozdata.firefox_desktop.newtab` t
-  CROSS JOIN
-    UNNEST(t.events) s
-  WHERE
-    date(t.submission_timestamp) = @submission_date
+    (
+      SELECT AS STRUCT
```
[the remark this answers is not in the document]
Comment (skahmann3) [reply sql/moz-fx-data-shared-prod/telemetry_derived/sponsored_tiles_clients_daily_v1/query.sql:9]: There might be a better way of doing this, IIRC I had errors when trying to sum directly

File: sql/moz-fx-data-shared-prod/telemetry_derived/sponsored_tiles_clients_daily_v1/query.sql [hunk 3]
```diff
@@ -1,93 +1,166 @@
...
+    normalized_country_code AS country,
+    locale,
+    normalized_channel,
+    normalized_os,
+    sample_id,
     COUNTIF(
       event = 'BLOCK'
       AND value LIKE '%spoc%'
-      AND source = 'TOP_SITES'
+      AND SOURCE = 'TOP_SITES'
```
[the remark this answers is not in the document]
Comment (skahmann3) [reply sql/moz-fx-data-shared-prod/telemetry_derived/sponsored_tiles_clients_daily_v1/query.sql:54]: It looks like it's lower case in the table, not sure if this is case sensitive?

File: sql/moz-fx-data-shared-prod/telemetry_derived/sponsored_tiles_clients_daily_v1/query.sql (hunk 1 above)
Comment (skahmann3) [reply sql/moz-fx-data-shared-prod/telemetry_derived/sponsored_tiles_clients_daily_v1/query.sql:261]: Or just add it to [unified metrics](https://github.com/mozilla/bigquery-etl/issues/3095)!

**Pull request merged**